index.html

```diff
@@ -5,12 +5,85 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Bangers|Caveat|Courgette|Lobster&display=swap" rel="stylesheet">
     <title>Responsive Cake webpage</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
     <!-- Add a link to your css file here -->
 </head>
 
 <body>
-    <!-- Add your markup here -->
+ <main>
+   <header>
```

Comment:
Good job for remembering to use the header and main elements, however the header html element should live outside of the main eg.
```
<header>
    ...
</header>
<main>
    ...
</main>
<footer>
    ....
</footer>
```